src/components/ikoner/IngenSoknaderFunnet.tsx

```diff
@@ -5,12 +5,12 @@ class IngenSoknaderFunnet extends React.Component {
         return (
             <svg
                 xmlns="http://www.w3.org/2000/svg"
-                width="73"
-                height="83"
+                width="5rem"
+                height="5rem"
                 viewBox="0 0 73 83"
```

Comment:
Skal viewBox fortsatt ha de gamle verdiene?

Reply:
Ja, viewbox må være sånn svgen var da den ble laget for å få riktig proporsjoner!

Reply:
Flottings.